### ISSUE TITLE
Failing unit test

### ISSUE THREAD

**Issue by dmitrizagidulin** [digitalcredentials/vc-status-list#6]
Couple of issues:
* [ ] Under Node 18-20, unit tests do not run (complain about the test files .js extension, demand "type": "module" in package.json
* [ ] (If you add type: module to get em to run) Unit test fails with:
```
      AssertionError: expected 'H4sIAAAAAAAAA-3OMQ0AAAgDsOHfNB72EJJWQ…' to equal 'H4sIAAAAAAAAA-3OMQ0AAAgDsElHOh72EJJWQ…'
      + expected - actual

      -H4sIAAAAAAAAA-3OMQ0AAAgDsOHfNB72EJJWQRMAAAAAAIDWXAcAAAAAAIDHFrc4zDzUMAAA
      +H4sIAAAAAAAAA-3OMQ0AAAgDsElHOh72EJJWQRMAAAAAAIDWXAcAAAAAAIDHFvRitn7UMAAA
```

This has to do with the breaking change this library had from 6.0 -> 7.0 (changed the endian-ness of bits inside the bitstring)

**Comment by kayaelle** [comment]
Possible related issue: https://github.com/digitalcredentials/vc-status-list/issues/6

**Comment by kezike** [comment]
@dmitrizagidulin I am still investigating, but I highly suspect that this issue is coming from the fact that our fork of DB's `bitstring` dependency uses big endian, while they have since updated theirs to use little endian, per this commit from Jan. 8: https://github.com/digitalcredentials/bitstring/commit/7d9e3f0af81f9631a6d292e7a0c97e6f11bd36d1. This is yet another example of where maintaining these dependency forks can get tricky 😅

**Comment by kezike** [comment]
I just confirmed that this was the issue. Technically, endianness does not matter for the spec, but since we synced the `vc-status-list` library with DB's main branch and that code defaults to using little endian when invoking the new `bitstring` code, we will need to update that dependency. I can work on this next.

**Comment by kezike** [comment]
@dmitrizagidulin Here are three PRs that fix this issue based on my previous comments:

- https://github.com/digitalcredentials/vc-status-list/pull/7
- https://github.com/digitalcredentials/bitstring/pull/4
- https://github.com/digitalcredentials/base64url-universal/pull/4

**Comment by dmitrizagidulin** [comment]
This has now been fixed in `v7.2.0`, closing.
